oam/install_scripts/mcs-loadbrm.py

```diff
@@ -2,17 +2,19 @@
 
 import configparser
 import subprocess
+import sys
 import xml.etree.ElementTree as ET
 
 sm_config = configparser.ConfigParser()
 sm_config.read('/etc/columnstore/storagemanager.cnf')
 cs_config = ET.parse('/etc/columnstore/Columnstore.xml')
 config_root = cs_config.getroot()
 
-storage = sm_config['ObjectStorage']['service']
-region = sm_config['S3']['region']
-bucket = sm_config['S3']['bucket']
-loadbrm = '/usr/bin/load_brm /var/lib/columnstore/data1/systemFiles/dbrm/{0}'
+storage = sm_config.get('ObjectStorage', 'service', fallback='LocalStorage')
+region = sm_config.get('S3', 'region', fallback='some_region')
+bucket = sm_config.get('S3', 'bucket', fallback='some_bucket')
+dbrmroot = config_root.find('./SystemConfig/DBRMRoot').text
```

Comment:
Getting dbrmroot dynamically. Note: didn't switch to using lxml as i don't think adding a new dependency is a good idea this late in the game

Reply:
OK, let's do this after the release.